docs/ae_api_ref.md

```diff
@@ -126,8 +126,8 @@ Executes a block of code, replacing named parameters with their provided values.
 
 The expected POST body is a JSON dictionary with two elements:
```

Comment:
Is there a difference between saying JSON dictionary and JSON object?  If there is no difference then I have never heard it referred to as a JSON dictionary.  Might just be me though...

Reply:
Yeah good question - _dict_ is the terminology for python specifically: https://www.w3schools.com/python/python_dictionaries.asp. And will probably be the more common runtime language for an analytics extension.